blockchain/indexers/cfindex.go

```diff
@@ -255,6 +255,37 @@ func (idx *CfIndex) DisconnectBlock(dbTx database.Tx, block *btcutil.Block,
 	return nil
 }
 
+// PruneBlock is invoked when an older block is deleted after it's been
+// processed.  For cfindx, PruneBlock is exactly the same as disconnecting a block.
+// TODO (kcalvinalvin): Consider keeping the filters at a later date to help with
+// reindexing as a pruned node.
+//
+// This is part of the Indexer interface.
+func (idx *CfIndex) PruneBlock(dbTx database.Tx, blockHash chainhash.Hash) error {
+	for _, key := range cfIndexKeys {
```

Comment:
We can do a small refactor here to allow this inner function to take the block hash directly, then we can call it for `DisconnectBlock` as well.